src/main.ts

```diff
@@ -167,4 +171,60 @@ async function releasesNodeModulesCheck(repository: string, ownername: string, s
     console.log(err);
   }       
 }
+async function vulnerabilityBotCheck(repository: string, ownername: string, secret_token: string, octokit: Octokit) {
+  try{
+    const result = await octokit.request('GET /repos/{owner}/{repo}/vulnerability-alerts',{
+      repo: repository,
+      owner: ownername,
+      headers : { 
+        Authorization: 'Bearer ' + secret_token,
+      },
+      mediaType: {
+        previews: [
+          'dorian'
+        ]
+      }
+    }); 
+    if(result.data === 'undefined'){
```

Comment:
If the output is "undefined", then the BOT is enabled?🤔

Reply:
Yes :p it shows Status: 204 No Content when it is enabled and Status: 404 Not Found when disabled

Reply:
should we have condition on return status code/value?
As even if due to some reason the API call fails, the response will be undefined and it will be call it as a success scene

Reply:
No no the response wont be undefined in that case

Reply:
It's better if we can have a condition on some concrete value, instead of "undefined"

Reply:
Yes I changed it to status matching